src/CaptureClient/include/CaptureClient/ApiEventIdProvider.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_CLIENT_API_EVENT_ID_PROVIDER_H_
+#define CAPTURE_CLIENT_API_EVENT_ID_PROVIDER_H_
+
+#include <cstdint>
+
+#include "ClientData/TimerTrackDataIdManager.h"
+#include "GrpcProtos/capture.pb.h"
+
+namespace orbit_capture_client {
+
+// The inferface defines a map from timer_info to ids. When called twice of identical `timer_info`
```

Comment:
I would write `TimerInfo` instead of `timer_info` as we are talking about the type and not a specific variable (like in the comment on `NameEqualityApiEventIdProvider`).